src/client.cpp

```diff
@@ -22,10 +22,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <sstream>
 #include <IFileSystem.h>
 #include "jthread/jmutexautolock.h"
+#include "util/auth.h"
```

Comment:
so you mean in src itself? Its so cramped there... network isn't good either because its not low level network related (like networkpacket) nor spanning a wide range of topics (like clientpackethandler).